python/ray/util/actor_pool.py

```diff
@@ -213,3 +213,64 @@ def _return_actor(self, actor):
         self._idle_actors.append(actor)
         if self._pending_submits:
             self.submit(*self._pending_submits.pop(0))
+
+    def has_free(self):
+        """Returns whether there are any idle actors available.
+
+        Returns:
+            True if there are any idle actors and no pending submits.
+
+        Examples:
+            >>> a1 = Actor.remote()
+            >>> pool = ActorPool(a1)
+            >>> pool.submit(lambda a, v: a.double.remote(v), 1)
+            >>> print(pool.has_free())
+            False
+            >>> print(pool.get_next())
+            2
+            >>> print(pool.has_free())
+            True
+        """
+        return len(self._idle_actors) > 0 and len(self._pending_submits) == 0
+
+    def pop(self):
```

Comment:
Can we rename this `pop_idle`?

Reply:
Sure thing! I agree this makes the function more clear. Just changed!